### ISSUE TITLE
[MNG-5041] Scope of dependecies defined in Dependecy Management can't be overriden

### ISSUE THREAD

**Issue by jira-importer** [apache/maven#7067]
**[Emmanuel Hugonnet](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=ehsavoie)** opened **[MNG-5041](https://issues.apache.org/jira/browse/MNG-5041?redirect=false)** and commented

If I define a dependency with in a parent pom using dependency management I can't override the scope of the dependency management declaration in a child pom.
If I use the dependency plugin the tree is as expected (the child override the parent) but if I use the effective-pom I get what is declared in the dependency management.
This seems a Maven 3.0.x bug

---

**Affects:** 3.0.2, 3.0.3

**Attachments:**
- [MNG-5041.zip](https://issues.apache.org/jira/secure/attachment/12713156/MNG-5041.zip) (_1.54 kB_)

3 votes, 2 watchers

**Comment by jira-importer** [comment]
**[Benjamin Bentmann](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=bentmann)** commented

Using this example project yields:

```
>mvn help:effective-pom -V
Apache Maven 3.0.3 (r1075438; 2011-02-28 18:31:09+0100)

  <dependencyManagement>
    <dependencies>
      <dependency>
        <groupId>junit</groupId>
        <artifactId>junit</artifactId>
        <version>3.8.2</version>
        <scope>provided</scope>
      </dependency>
    </dependencies>
  </dependencyManagement>
  <dependencies>
    <dependency>
      <groupId>junit</groupId>
      <artifactId>junit</artifactId>
      <version>3.8.2</version>
      <scope>compile</scope>
    </dependency>
  </dependencies>
```

i.e. the scope from the parent's dependency management was successfully overriden by the child.

**Comment by jira-importer** [comment]
**[Arnaud Heritier](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=aheritier)** commented

Benjamin did you try with the depMgt in a parent pom and the dep in a child ? It is how I understand the explained testcase.
Emmanuel, can you provide a little testcase to reproduce it ?

**Comment by jira-importer** [comment]
**[Emmanuel Hugonnet](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=ehsavoie)** commented

Yes, Silverpeas poms on github ;)
)After more investigations using the latest help plugin and effective pom all is working fine.
It seems to come from a bug in the assembly plugin (2.2beta5) which was fixed in 2.2.1.
The fact that the dependency plugin display wrong informations lead us to creating this bug.
Sorry for the inconvenience.
